graph-builder/src/registry.rs

```diff
@@ -121,17 +127,35 @@ pub fn fetch_releases(
     let tagged_layers = thread_runtime.block_on(tags)?;
 
     let mut releases = Vec::with_capacity(tagged_layers.len());
-    for (tag, layer_digests) in tagged_layers {
-        let release = cache_release(
+    for (tag, manifestref, layer_digests) in tagged_layers {
+        if let Some(release) = cache_release(
             layer_digests,
             authenticated_client.to_owned(),
             registry_host.to_owned(),
             repo.to_owned(),
             tag.to_owned(),
             cache,
-        )?;
-        if let Some(metadata) = release {
-            releases.push(metadata);
+        )? {
+            let manifestref = manifestref.as_str();
+            let quay_labels = thread_runtime.block_on(quay_client.get_labels(
+                &repo,
+                &manifestref,
+                Some(&quay_label_filter),
+            ))?;
+
+            let release = Release {
+                metadata: Metadata {
+                    metadata: quay_labels
+                        .into_iter()
+                        .map(|label| (label.key, label.value))
+                        .chain(release.metadata.metadata)
```

Comment:
I think this chaining is still in the wrong order, see https://github.com/openshift/cincinnati/pull/48#discussion_r251196071.

Did you maybe pick up an older version of your commits? The other PR seems updated and fine.

Reply:
Thanks, I must've lost that change along the way, it's back in.